src/tribler/core/upgrade/version_manager.py

```diff
@@ -121,6 +136,10 @@ def calc_state_size(self):
                 result += f.stat().st_size
         return result
 
+    def upgrade_size(self, reserved_space=RESERVED_STORAGE) -> int:
```

Comment:
**nitpick:** It is better to rename the method to something like `get_upgrade_size`; otherwise, `upgrade` sounds like a verb, and the method name can be read as if it modifies some "size".